src/app/containers/hooks/useSubmitMovieFormik.js

```diff
@@ -0,0 +1,20 @@
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+const UseSubmitMovieFormik = (onSubmit) => {
```

Comment:
This hook should start with lower case `useSubmitMovieFormik`

Reply:
hook was deleted but point noted